packages/material-ui-styles/test/index.spec.tsx

```diff
@@ -129,10 +125,13 @@ function testGetThemeProps(theme: Theme, props: AppBarProps): void {
     defaultTheme: invalidCustomTheme,
   });
 
-  // $ExpectError
-  makeStyles<typeof style>(style, {
```

Comment:
We should keep this test and simply add that this is undesired type behavior. Haven't made up my mind yet if this is more valueable than this specific auto-completion.

Reply:
How can we keep the test when the generic signature has changed though?

Reply:
I mean, the previous test (with inferred generics) covers the same scenario, the only difference is that generics are not passed with. But what is the scenario for when you want to pass the `typeof style` as a parameter explicitly?

Reply:
Right. The UI wasn't showing the previous test so I assumed we lost this behavior. I still fall for the github UI sometimes. Really need to force myself to review these in vscode instead of github ui.